settings/mailutils.setting.php

```diff
@@ -12,4 +12,16 @@
     'is_contact'  => 0,
     'description' => ts('This determines the location type used for new email addresses added by mailutils.'),
   ],
+  'mailutils_case_assignment_email_blacklist' => [
+    'name'        => 'mailutils_case_assignment_email_blacklist',
+    'type'        => 'String',
+    'serialize'   => CRM_Core_DAO::SERIALIZE_JSON,
```

Comment:
Random core rant:
I've previously been using `'type' => 'Array'` without `serialize` for JSON settings, but docs discourage this.

What's annoying is that the recommended way to interact with settings - i.e. `Civi::settings()->get()` does not seem to care about `serialize` and just returns a JSON string you have to decode yourself. Or you go through API4 (as is done here) which deals with `serialize` correctly.

Guess I'll just be using API4 exclusively to interact with settings in the future. 🤷